docs/src/dataDisplay/telephoneLink/telephoneLink.jsx

```diff
@@ -1,15 +1,52 @@
 import {
+    Header,
+    TelephoneLink,
     Typography,
 } from 'react-cm-ui';
 import React from 'react';
 import ComponentApi from '../../global/componentApi';
 import Heading from '../../global/heading';
+import Highlighter from '../../global/highlighter';
 import MarkdownContainer from '../../global/markdownContainer';
 import Main from '../../global/main';
 /* eslint-disable import/no-named-default, import/extensions */
 import { default as telephoneLinkDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/dataDisplay/telephoneLink/telephoneLink';
 /* eslint-enable import/no-named-default, import/extensions */
 
+const idSample = 'myId';
+const numberSample = '525512345678';
+const formattedNumberSample = 'MX';
+const classSample = 'myClass';
+
+const telephoneLinkSample = `import React from 'react';
+
+import Loader from '../app/Elements/Loader.react';
+
+export default class TypeSample extends React.Component {    
```

Comment:
`TypeSample` is a weird function name here (likely copied and pasted).

Reply:
Fixed.

Reply:
Looks good.  Resolving.